tests/admin_scripts/management/commands/with_suppress_command.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+
+    help = 'Test basic commands with suppress'
+    requires_system_checks = []
+    suppressed_base_arguments = {'--verbosity', '--traceback', '--settings', '--pythonpath', '--no-color',
+                                 '--force-color', '--version'}
+
+    def add_arguments(self, parser):
+        parser.add_argument('args', nargs='*')
+
+    def handle(self, *labels, **options):
+        print('EXECUTE:WithSuppressCommand labels=%s, options=%s' % (labels, sorted(options.items())))
```

Comment:
`args` and `labels` are not related with this patch. We can remove them:
```suggestion

    def handle(self, *labels, **options):
        print('EXECUTE:WithSuppressCommand options=%s' % sorted(options.items()))
```